Adyen/UI/Form/Items/SelectableFormItem/SelectableFormItemView.swift

```diff
@@ -0,0 +1,179 @@
+
+        return imageView
+    }()
+
+    // MARK: - Content StackView
+
+    private lazy var contentStackView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: [imageView, titleLabel, checkmarkImageView])
+        stackView.setCustomSpacing(16, after: imageView)
+        stackView.spacing = 8
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.preservesSuperviewLayoutMargins = true
+        stackView.isLayoutMarginsRelativeArrangement = true
+        stackView.setContentHuggingPriority(.required, for: .vertical)
+        stackView.axis = .horizontal
+        stackView.alignment = .center
+        stackView.distribution = .fill
+
+        return stackView
+    }()
+
+    // MARK: - Item Button
+
+    private lazy var itemButton: UIButton = {
+        let customButton = UIButton(type: .custom)
+        customButton.addTarget(self, action: #selector(didSelectItemButton), for: .touchUpInside)
+        customButton.preservesSuperviewLayoutMargins = true
+        customButton.translatesAutoresizingMaskIntoConstraints = false
+        customButton.accessibilityIdentifier = item.identifier.map {
+            ViewIdentifierBuilder.build(scopeInstance: $0, postfix: "button")
+        }
+
+        customButton.addSubview(contentStackView)
+        contentStackView.isUserInteractionEnabled = false
+        contentStackView.adyen.anchor(inside: customButton)
+
+        return customButton
+    }()
+
+    @objc internal func didSelectItemButton() {
+        item.selectionHandler?()
+    }
+
+    /// Initializes the selectable form item view.
+    public required init(item: SelectableFormItem) {
+        super.init(item: item)
+        backgroundColor = item.style.backgroundColor
+
+        addSubview(itemButton)
+
+        updateItemData(item: item)
+        updateImageView(style: item.style)
+        configureConstraints()
+
+        preservesSuperviewLayoutMargins = true
+
+        itemButton.adyen.anchor(inside: self.layoutMarginsGuide)
+
+        observe(item.$isSelected) { [weak self] isSelected in
+            guard let self else { return }
+            self.checkmarkImageView.isHidden = !isSelected
+        }
+    }
+
+    override public func didMoveToWindow() {
+        super.didMoveToWindow()
+        updateIcon()
+    }
+
+    override public func layoutSubviews() {
+        super.layoutSubviews()
+
+        imageView.adyen.round(using: item.style.imageStyle.cornerRounding)
+    }
+
+    private func updateImageView(style: SelectableFormItemStyle) {
+        imageView.contentMode = style.imageStyle.contentMode
+        imageView.clipsToBounds = style.imageStyle.clipsToBounds
+        imageView.layer.borderWidth = style.imageStyle.borderWidth
+        imageView.layer.borderColor = style.imageStyle.borderColor?.cgColor
+    }
+
+    private func updateItemData(item: SelectableFormItem) {
```

Comment:
This function became redundant since item is not optional here as in ListItemView.
I think we can set every area that needs `item` during each of those properties lazy initilization.

Reply:
70e58402066b124f6f066f9753e7e9533ab45ff6